jdaviz/configs/cubeviz/helper.py

```diff
@@ -147,7 +151,8 @@ def get_data(self, data_label=None, cls=None, subset_to_apply=None, function=Non
         """
         if function is True:
             return self.specviz.get_data(data_label=data_label, cls=cls,
-                                         subset_to_apply=subset_to_apply)
+                                         subset_to_apply=subset_to_apply,
+                                         spectral_to_spatial=spectral_to_spatial)
```

Comment:
could we instead call `specviz._get_data` so that the public `get_data` doesn't have to have the cubeviz-specific kwarg exposed?

Reply:
Looks like I would have to move `function` logic or redo it to use the `specviz._get_data` call. Is that worth it to not have to add cubeviz-specific kwargs?

Reply:
I think it would be worth having that `function = getattr(default_sp_viewer.state, 'function', None)` moved or duplicated to avoid the cubeviz-specific kwargs in a specviz method.

I also find `spectral_to_spatial` to be confusing.  This is the spatial subset that will be used to extract/collapse the cube to a 1d spectrum and _then_ apply a spectral `subset_to_apply` as a mask, right?

Reply:
I believe all of these review comments have been addressed.